lib/oasis/base_adapter.js

```
@@ -85,7 +85,12 @@ BaseAdapter.prototype = {
 
       oasis.configuration.oasisURL = event.data.oasisURL;
 
-      adapter.loadScripts(event.data.base, event.data.scriptURLs, oasis);
+      try {
+        adapter.loadScripts(event.data.base, event.data.scriptURLs, oasis);
+      } catch( e ) {
+        // The required resources can't be loaded, abort
```

probably should not suppress errors here